frontend/Store.js

```diff
@@ -288,13 +288,21 @@ class Store extends EventEmitter {
       this.emit('hover');
       this._bridge.send('highlight', id);
     } else if (this.hovered === id) {
-      this.hovered = null;
-      this.emit(id);
-      this.emit('hover');
-      this._bridge.send('hideHighlight');
+      this.hideHighlight();
     }
   }
 
+  hideHighlight() {
+    this._bridge.send('hideHighlight');
```

Comment:
Yeah -- [here's the file](https://github.com/facebook/react-devtools/blob/devtools-next/frontend/Highlighter/setup.js) used to connect a `Highlighter` to an `Agent`